frontend/src/App.js

```diff
@@ -41,13 +41,15 @@ import InputText from "./components/InputText";
 import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
```

Comment:
no space after '{' or before '}'

Reply:
In React's docs, they put spaces before and after https://create-react-app.dev/docs/importing-a-component/